### PR TITLE
Added support for es6 "import"

### PR DESCRIPTION
**Pull request by kuashe**
Added support for es6 "import"

### CODE REVIEW

**Comment by kuashe** [comment]
@GianlucaGuarini @aMarCruz Please check it . How do I solve code coverage decrease ?

File: src/core.js [hunk 1]
```diff
@@ -1080,7 +1115,7 @@ function compile (src, opts, url) {
       }
 
       // replace the tag with a call to the riot.tag2 function and we are done
-      return mktag(tagName, html, styles, attribs, jscode, opts)
+      return mktag(tagName, html, styles, attribs, jscode, opts, imports)
```
Comment (GianlucaGuarini) [inline src/core.js]: @aMarCruz maybe we can use an option object here.. there are too many arguments here

**Comment by GianlucaGuarini** [comment]
@aMarCruz please verify this pull request and we can release the riot-compiler@2.5.0

File: src/core.js (hunk 1 above)
Comment (GianlucaGuarini) [inline src/core.js]: I would also prefer to have the opts object always as last argument. Could you please fix it @kuashe ?

**Pull request merged**